topics/list_topics_test.go

```diff
@@ -0,0 +1,120 @@
+// Package topics is an experimental package that provides additional tooling
+// around Kafka Topics. This package does not make any promises around
+// backwards compatibility.
```

Comment:
We don't need to repeat the package comment

Reply:
Done.